### PR TITLE
Update Polygon.html

### PR DESCRIPTION
**Pull request by stolstov**
@alocke I wanted to clarify the Esri valid polygon. Could you review?

### CODE REVIEW

File: doc/Polygon.html [hunk 1]
```diff
@@ -64,10 +64,19 @@ <h2 id="boundary">Boundary and Rings</h2>
...
-			classified unambiguously as either in the exterior, in the interior or on the
-			boundary of the polygon. A valid polygon is said to be <i>simple</i>.
+			<p>A valid polygon has the following main property:
+				<li>for every polygon segment, the polygon exterior is unambiguously to the left of the segment and
+					the polygon interior is unambiguously to the right of the segment.</li>
+				As a result of the above property the following is also true:
+				<li>segments can touch other segments only at the end points,</li>
+				<li>segments have non-zero length,</li>
+				<li>extrior rings are clockwise and holes are counterclockwise,</li>
+				<li>each polygon ring has non-zero area.</li>
```
[the remark this answers is not in the document]
Comment (stolstov) [reply doc/Polygon.html]: Removed

[the remark this answers is not in the document]
Comment (stolstov) [reply doc/Polygon.html]: Could you continue to review?

File: doc/Polygon.html [hunk 2]
```diff
@@ -64,12 +64,20 @@ <h2 id="boundary">Boundary and Rings</h2>
...
-			at vertices, no dangling segments and, in general, an arbitrary point can always be
-			classified unambiguously as either in the exterior, in the interior or on the
-			boundary of the polygon. A valid polygon is said to be <i>simple</i>.
+			<p>A valid polygon has the following main property:
+				<li>for every polygon segment, the polygon exterior is unambiguously to the left of the segment and
+					the polygon interior is unambiguously to the right of the segment.</li>
+				As a result of the above property the following is also true:
+				<li>segments can touch other segments only at the end points,</li>
+				<li>segments have non-zero length,</li>
+				<li>exterior rings are clockwise and holes are counterclockwise,</li>
```
Comment (alocke) [inline doc/Polygon.html]: Can you change "exterior" to "outer" just because you talk about the polygon exterior above and it might be confusing.

Comment (stolstov) [reply doc/Polygon.html]: Done.

**Pull request merged**